app/assets/javascripts/account/AccountService.js.coffee

```diff
@@ -51,6 +51,26 @@ AccountService = ($state, $auth, $modal, $http, $translate, ShortFormApplication
         return false
       )
 
+  Service.requestPasswordReset = ->
+    params =
+      email: Service.userAuth.email
+    $auth.requestPasswordReset(params).then((resp) ->
+      Service.userAuth.resetPwdEmailSent = true
+    ).catch (resp) ->
+      # handle error response
+    return
+
+  Service.updatePassword = ->
+    params =
+      password: Service.userAuth.password
+      password_confirmation: Service.userAuth.password_confirmation
+
+    $auth.updatePassword(params).then((resp) ->
+      $state.go('dahlia.my-applications')
```

Comment:
@kaptron yeah in the story it says to be directed to `my-applications`. See:

```
'update password' button is built but non-functional (becomes functional when 'my applications' page is built since that is where it will take the user) -
```

Although I tend to agree with you. I would prefer `my-account`.

Re userAuth namespacing: sounds good -- I will the reset password stuff to use `userAuth.user` :dancer: